bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/RequestStats.java

```diff
@@ -92,6 +93,14 @@ public class RequestStats {
         parent = ADD_ENTRY_REQUEST
     )
     private final OpStatsLogger addEntryStats;
+
+    @StatsDoc(
+            name = WRITE_THREAD_QUEUED_LATENCY,
+            help = "operation stats of enqueuing requests to write threadpool queue",
+            parent = ADD_ENTRY_REQUEST
+    )
+    private final OpStatsLogger writeThreadQueueStats;
```

Comment:
writeThreadQueuedStats ?

Reply:
To calculate the time a request spends in the write thread pool queue.

Reply:
rename to writeThreadQueuedLatency.

PTAL,thanks! @hangc0276